src/Entity/Carrinho.php

```diff
@@ -188,7 +188,8 @@ public function jsonSerialize(): array
             'status' => $this->status,
             'valor_total'  => $this->valor_total,
             'criado_em' => $this->criado_em->format("d/m/Y"),
-            'finalizado_em' => $this->finalizado_em,   
+            'finalizado_em' => $this->finalizado_em,
+            'atualizado_em' => $this->atualizado_em,
```

Comment:
faltou formatar igual o `criado_em`
```suggestion
            'finalizado_em' => $this->finalizado_em?->format("d/m/Y"),
            'atualizado_em' => $this->atualizado_em?->format("d/m/Y"),
```